### PR TITLE
Feature/new meta editor

### PR DESCRIPTION
**Pull request by wingechr**

### CODE REVIEW

**Comment by MGlauer** [comment]
@wingechr Can I review this now? It is still marked as WIP.

**Comment by wingechr** [comment]
@MGlauer yes

File: dataedit/templates/dataedit/dataview.html [hunk 1]
```diff
@@ -117,14 +116,21 @@ <h4>API Usage</h4>
...
+  <hr>
+
   <h4>
     Meta Information
   </h4>
-  <div class="btn-group btn-group-sm">
-    <button class="btn btn-success" onclick="downloadMetadata();">Download JSON</button>
+  <div class="mb-2">
+    <button class="btn btn-sm btn-success" onclick="downloadMetadata();">Download JSON</button>
+    <a href="{{table}}/meta_edit" type="button" class="btn btn-success btn-sm ">Edit</a>
```
Comment (MGlauer) [inline dataedit/templates/dataedit/dataview.html:131]:
```suggestion
    <a href="{{table}}/meta_edit" type="button"  class="btn btn-success btn-sm {% if not can_add %} disabled {% endif %}">Edit</a>
```

File: dataedit/templates/dataedit/dataview.html [hunk 2]
```diff
@@ -117,14 +116,21 @@ <h4>API Usage</h4>
...
+  </div>
+  <hr>
+
   <h4>
     Meta Information
   </h4>
-  <div class="btn-group btn-group-sm">
-    <button class="btn btn-success" onclick="downloadMetadata();">Download JSON</button>
+  <div class="mb-2">
+    <button class="btn btn-sm btn-success" onclick="downloadMetadata();">Download JSON</button>
```
Comment (MGlauer) [inline dataedit/templates/dataedit/dataview.html:130]:
```suggestion
    <button class="btn btn-sm btn-success" onclick="downloadMetadata();" data-toggle="tooltip" title="Download metadata in JSON format">Download JSON</button>
```

File: dataedit/templates/dataedit/dataview.html [hunk 3]
```diff
@@ -117,14 +116,21 @@ <h4>API Usage</h4>
...
+    <a href="{{ table }}/permissions" type="button" class="btn btn-danger">Permissions</a>
+  </div>
+  <hr>
+
   <h4>
     Meta Information
   </h4>
-  <div class="btn-group btn-group-sm">
-    <button class="btn btn-success" onclick="downloadMetadata();">Download JSON</button>
+  <div class="mb-2">
```
Comment (MGlauer) [inline dataedit/templates/dataedit/dataview.html:129]:
```suggestion
  <div class="mb-2" {% if not can_add %} data-toggle="tooltip" title="You need write permissions on this table to edit metadata" {% endif %}>
```

**Review by MGlauer — CHANGES_REQUESTED** [review]
The edit button works even if the user can not edit the metadata. We should disable it in this case and show a tooltip explaining the situation. But tooltip don't work on disabled inputs - so we have to add it to the top level element and add a tooltip to the other button to prevent the same tooltip showing on both buttons.

File: dataedit/templates/dataedit/dataview.html (hunk 2 above)
Comment (MGlauer) [reply dataedit/templates/dataedit/dataview.html:130]: We can add the `not can_edit` conditional here, too. But I feel that a tooltip could improve this button in general.

**Review by MGlauer — APPROVED** [review]

**Pull request merged**